interview-questions.md

```diff
@@ -7,60 +7,64 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is a kind of version control used by us developers. The difference between Git and GitHub is that Git is the software that tracks changes made by the user and GitHub stores them.
 
-  Researched answer:
+  Researched answer: Git is a widely used version control system that allows for its users to manage and track their history. GitHub is a host that manages the Git repositories. Another difference is that GitHub is cloud-based.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: Boolean values are either true or false. Therefore, the types of evaluations that return a Boolean value must
 
-  Researched answer:
+  Researched answer: Both equality operators and logical OR operators return a Boolean value.
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: The difference between the index and the value is that the index will tell us the position and the value tells us the number of.
 
+  Researched answer: I couldn't find much doing internet research, but I have in my notes from lecture with Austin that the index tells us the position while the
+  value will tell us the number of or how many.
 
 
 4. What is a built-in method?
 
-  Your answer:
+  Your answer: A built-in method is often times associated with its own lengthy amount of code and later added to the language to effectively solve an problem.
 
-  Researched answer:
+  Researched answer: A built-in method is in place to simplify the life of the user. It is a function that is built-in to a language and/or application.
 
 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-  Your answer:
+  Your answer: An efficient pair programmer uses proper time to trade roles-- no more than 25 minutes.
+  The driver does not chime in on what the navigator is asking them to code, but follows directions. The navigator pays attention to what the driver is
+  typing and is responsible for catching errors.
 
-  Researched answer:
+  Researched answer: The driver should write the code according to the navigators direction, actively listen to instructions provided, trust the navigator, and ask questions
+  for clarification. The navigator controls the code that is to be written, clearly communicates, check for errors, and explains why they have chosen a certain solution.
```

Comment:
Very good, with the small adjustment that the Driver should control the code that is written. The navigator is there for all the other stuff you mentioned.